action_tutorials/README.md

```diff
@@ -0,0 +1,24 @@
+# ROS2 Action Tutorials
+
+This tutorial demonstrates implementing ROS action servers and action clients.
+
+The action server in this case takes in an integer value between 0 and 45 named *order* and returns the Fibonacci sequence, a sequence with the form:
+$$F_0 = 0$$
+$$F_1 = 1$$
+$$F_{order}=F_{order-1} + F_{order-2}$$
+
+The action server calculates each number in the sequence one at a time and at returns a partial sequence as feedback at each iteration. 
+
+If the action is cancelled before the entire sequence is calculated, a partial sequence will be returned.
+
+The action client in this tutorial sends a goal to the action server with an order of 10. It logs each partial sequence returned as feedback. 
```

Comment:
```suggestion
The action client in this tutorial sends a goal to the action server with an order of 10.
It logs each partial sequence returned as feedback. 
```

In general, having the sentences on different lines makes it easier to see what has changed in diffs and to give specific feedback.